pkg/controller/account/secrets.go

```diff
@@ -0,0 +1,47 @@
+package account
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// SRESecretInput is a struct that holds data required to create a new secret CR for SRE admins
+type SRESecretInput struct {
+	SecretName, NameSpace, awsCredsSecretIDKey, awsCredsSecretAccessKey, awsCredsSessionToken, awsCredsConsoleLoginURL string
+}
+
+func (input SRESecretInput) newSTSSecret() *corev1.Secret {
```

Comment:
Is this function still used?

Reply:
I plan on taking out this function.  I just wanted to make sure it was alright to abstract away the individual implementations of the secret structs and functions into a single one that just takes the map before I finished up removing all of the other secret references.